onedal/_device_offload.py

```diff
@@ -190,9 +190,9 @@ def wrapper_impl(*args, **kwargs):
                     result = _convert_to_dpnp(result)
                 return result
 
-        if not get_config().get("transform_output"):
+        if get_config().get("transform_output") in ["default", None]:
```

Comment:
Tiny detail: It's more common to use a tuple in such cases.

```suggestion
        if get_config().get("transform_output") in ("default", None):
```